pkg/install/install.go

```diff
@@ -132,6 +132,15 @@ func Serving() error {
 
 	fmt.Println("    Core installed...")
 
+	if registries != "" {
+		configPatch := fmt.Sprintf(`{"data":{"registries-skipping-tag-resolving":"%s"}}`, registries)
+		ignoreRegistry := exec.Command("kubectl", "patch", "configmap", "-n", "knative-serving", "config-deployment", "-p", configPatch)
+		if err := runCommand(ignoreRegistry); err != nil {
+			return fmt.Errorf("tag resolving configuration: %w", err)
+		}
+		fmt.Println("    Tag resolving configuration patched...")
```

Comment:
I would add the *why* not so much the *what* or *how*. I.e. how it is patched, what does this change ?. Also not sure if the message itself might be too noisy and potentially confusing for a beginner who is running the quickstart. At the end it should _just work_, and a beginner should not worry about what a "tag resolving configuration" is.

Reply:
I'm happy to remove the comment if we want, it's hard to briefly explain the feature.

Reply:
Switched this to "enabling local registry deployments"